build.sh

```diff
@@ -18,6 +18,8 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
+RAPIDS_VERSION=23.12
```

Comment:
This will have to get updated again very soon.  Should we just add support for updating this in [update-version.sh](https://github.com/rapidsai/cugraph/blob/branch-23.12/ci/release/update-version.sh)?